src/main/java/payapal/Perfection.java

```diff
@@ -0,0 +1,42 @@
+package main.java.payapal;
+
+public class Perfection {
+
+    private static Perfection perf;
+
```

Comment:
Still talking about **Singleton**, it's a good practice to create a private constructor to prevent the class to be instantiated.